src/pages/Home/Home.page.jsx

```diff
@@ -31,7 +73,24 @@ function HomePage() {
         </>
       ) : (
         <Link to="/login">let me in →</Link>
-      )}
+      )} */}
+      {youtubeData.items.map((video) => (
+        <VideoCard key={video.etag}>
+          <img
+            className="videoCardMedia"
+            src={video.snippet.thumbnails.high.url}
+            alt={video.kind}
+          />
+          <div className="videoTextContainer">
+            <h3 className="videoCardTitle">{video.snippet.title}</h3>
+            <span className="videoCardDescription">
+              {video.snippet.description
+                ? video.snippet.description
+                : 'Este video no cuenta con descripción'}
```

Comment:
nice fallback 👍 I don't know if nullish coalescing operator can be used here like this, to avoid redundance:
```js
video.snippet.description ?? 'Este video no cuenta con descripción'
```

Reply:
Couldn't get it to work, but I used `video.snippet.description || 'Este video no cuenta con descripción'` instead to avoid redundance